src/components/timeline/scss/timeline-item-group.scss

```diff
@@ -1,7 +1,7 @@
 md-chip::part(chip){
   border: 1px solid var(--md-secondary-bg-color, #c7c7c7);
   margin-bottom: .25rem;
-  background-color: var(--md-tertiary-two-bg-color, #3b3b3b);
+  background-color: var(--md-tertiary-two-bg-color, #c7c7c7);
```

Comment:
why did you change this? 
--md-tertiary-two-bg-color = gray-80 or gray-10 and neither of those are #c7c7c7??